### DISCUSSION TITLE
Qiita CLI の publish コマンドを実行すると別のファイルが生成される

### DISCUSSION THREAD

**Discussion by jyoppomu** [increments/qiita-discussions#537] [要望・アイデア・不具合]
## Qiita CLI Version

v1.2.0

## 詳細

1. `npx qiita new newArticleBase` で新しいファイルを作る
2. `public/newArticleBase.md` を編集する
3. `npx qiita publish newArticleBase` で記事を投稿する
4. `public/newArticleBase.md` とは別に `public/{hashId}.md` が作られる

## 要望

最初に指定したベース名で管理したい。2ファイルあると管理ができない。
もしくは、`npx qiita new` を実行したときに記事の id を用いてファイル名を生成してほしい。

**Comment by tanaka-asami415** [comment]
@aYukiYoshida 
ご報告ありがとうございます！
別のファイルは作られないようになっているはずなのですが、一旦調査いたします🙇

**Comment by jyoppomu** [comment]
(コメントを投稿されたようですが、削除されてますね...)
https://github.com/increments/qiita-cli/pull/66 での対応ありがとうございます。
まだ、リリースには含まれていないようなので、待ちます。

**Reply by kimkim0814** [reply]
@aYukiYoshida
先ほど誤った内容を投稿してしまったため、一度削除させていただきました🙇

開発環境にてQiita CLI の publish コマンドを実行すると別のファイルが生成される不具合の方、確認できました。

ご確認頂いた通り、修正のPull Requestを作成しmergeしました。
リリースは来週連休明けに行う予定ですので、もうしばらくお待ちいただけると幸いです🙇

**Comment by kimkim0814** [comment]
@aYukiYoshida
お待たせいたしました!

Qiita CLI の publish コマンドを実行すると別のファイルが生成される不具合について、
修正を含むバージョンを v1.3.0  としてリリースしました！
https://github.com/increments/qiita-cli/releases/tag/v1.3.0

この度はご報告ありがとうございました。

**Comment by jyoppomu** [comment]
@kimkim0814 

修正版のリリースありがとうございます。
早速確認し、報告した不具合が再現しないことが確認できました。
